app/controllers/ops_controller/ops_rbac.rb

```diff
@@ -8,6 +8,14 @@ module OpsController::OpsRbac
     'Tenant'   => 'tenant'
   }.freeze
 
+  def role_allows?(**options)
+    if MiqProductFeature.my_root_tenant_identifier?(options[:feature]) && params.key?(:id)
+      options[:feature] = MiqProductFeature.tenant_identifier(options[:feature], params[:id].to_s.split("-").last)
```

Comment:
why the split?

According to the tests, this is always just the id, are the tests wrong?

Reply:
yes exactly I was looking for this, 👍 

this goes for button (if it should display "Manage Tenant Quotas" button in toolbar) and for controller actions.